demo/asset_section.js

```diff
@@ -196,11 +196,11 @@ shakaDemo.load = function() {
   // event.  This seems to work only because Shaka Player has already created a
   // MediaSource object and set video.src.
   var playPromise = shakaDemo.video_.play();
-  // Check if browser supports Media Session first.
-  if ('mediaSession' in navigator) {
-    // Reset the media session.
-    navigator.mediaSession.metadata = new MediaMetadata();
-    if (playPromise !== undefined) {
+  if (playPromise !== undefined) {
```

Comment:
Thanks for the PR! Small nit: we generally avoid using '===' operator in the project unless there's a good reason to have it instead of just '=='.